rules.ini

```diff
@@ -44,7 +44,16 @@ ChromeEngine[] = \.rpack$
 Clausewitz[] = (?:^|/)common/defines\.(?:lua|txt)$
 Clausewitz[] = ^(?:game/)?common/defines/00_defines\.txt$
 ClickTeamFusion = (?:^|/)mmfs2\.dll$
-Cocos2d = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)libcocos2d\.dll$
+Cocos[] = (?:^|/)Cocos2d\.lua$
+Cocos[] = (?:^|/)jsb-adapter/jsb-engine\.jsc?$
+Cocos[] = (?:^|/)jsb-adapter/engine-adapter\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.[0-9a-z]+\.jsc?$
+Cocos[] = (?:^|/)cocos2d-jsb\.jsc?$
+Cocos[] = (?:^|/)assets/main/cc\.config\.json$
+Cocos[] = (?:^|/)assets/main/cc\.config\.[0-9a-z]+\.json$
+Cocos[] = (?:^|/)cocos-js/cc\.jsc?$
```

Comment:
this one matches only one game and its demo, no worth to support imo 
or we can just look for the folder `cocos-js`
Depots that matched:
* [1995681 - ](https://steamdb.info/depot/1995681/)
* [2019441 - ](https://steamdb.info/depot/2019441/)

Reply:
It's ok for me to just check `cocos-js`, but I thought 
```
Cocos[] = (?:^|/)cocos-js/cc\.jsc?$
Cocos[] = (?:^|/)cocos-js/cc\.[0-9a-z]+\.jsc?$
```
regex were more precise.